drivers/periph_common/init.c

```diff
@@ -72,4 +75,8 @@ void periph_init(void)
 #ifdef MODULE_PERIPH_USBDEV
     usbdev_init_lowlevel();
 #endif
+
+#if defined(MODULE_PERIPH_FLASHPAGE) || defined(MODULE_PERIPH_FLASHPAGE_RAW)
```

Comment:
The problem now is that this is required for all cpu that provide this feature.

Would it make sense to call this STM32 specific function from the `cpu_init` of STM32 and only for F2/F4 ?

Reply:
@aabadie I had to change the way this was done, while testing for https://github.com/RIOT-OS/RIOT/pull/11682 I realized some thimes the configuration value is changed and it should be set for every erase/program operation